apps/merkle_patricia_tree/lib/merkle_patricia_tree/db/ets.ex

```diff
@@ -4,14 +4,12 @@ defmodule MerklePatriciaTree.DB.ETS do
   is backed by :ets.
   """
 
-  alias MerklePatriciaTree.{DB, Trie}
-
   @behaviour MerklePatriciaTree.DB
 
   @doc """
   Performs initialization for this db.
   """
-  @spec init(DB.db_name()) :: DB.db()
```

Comment:
whats the reasoning behind deleting specs?

Reply:
@InoMurko `@impl true` annotation checks if the following function is the implementation of a behaviour

https://github.com/elixir-lang/elixir/issues/5734

Reply:
how does this affect typespecs?

Reply:
it replaces typespecs in the implementation with those in the behaviour

Reply:
I'm actually a big fan of `@impl true` since it's possible that behaviors and functions get out of sync in regards to typespecs. This prevents that from happening.